threedi_schema/__init__.py

```diff
@@ -2,5 +2,5 @@
 from .domain import constants, custom_types, models  # NOQA
 
 # fmt: off
-__version__ = '0.222.3.dev0'
+__version__ = '0.223.dev1'
```

Comment:
similar to my comment the other day: if you need this then I am fine with this change otherwise I am a bit hesitant to change this

Reply:
reverted in cf88667d8fe26e2ca5a008ad3aa1a4868ce43d89